zb-classifier.js

```diff
@@ -469,15 +472,30 @@ class ZigbeeClassifier {
       'parseOnOffAttr',               // parseValueFromAttr
       CONFIG_REPORT_INTEGER
     );
+    if (endpoint.deviceId === '000b') {
+      // The ORVIBO Smart Relay doesn't seem to support config reports. It
+      // also doesn't seem to have a reasonable modelId (the report I got showed
+      // a modelId of 82c167c95ed746cdbd21d6817f72c593), so I'm not sure if
+      // that's real or not. It also showed a deviceId of 000b which doesn't
+      // seem to be documented in the zigbee spec, so for now, I'm using that
+      // to identify this. I have one on order and when it arrives I'll get a
+      // chance to play with it and see if the weird string of numbers really
+      // is a valid modelId.
+      property.fireAndForget = true;
+    }
+    return property;
   }
 
-  addButtonOnProperty(node, genOnOffOutputEndpoint) {
+  addButtonOnProperty(node, genOnOffOutputEndpoint, suffix) {
+    if (typeof suffix === 'undefined') {
+      suffix = '';
+    }
     const property = this.addProperty(
       node,                           // device
-      'on',                           // name
+      `on${suffix}`,                           // name
```

Comment:
nit: comment spacing should align with previous line

Reply:
done